src/datacatalog/authorization.py

```diff
@@ -110,6 +119,8 @@
                           security_requirements: T.Dict[
                               str, T.Optional[T.Iterable]
                           ]) -> bool:
+    if security_requirements is None:
```

Comment:
Als `security_requirements is None`, dan zit er een fout in de OpenAPI spec file. Dit zou nooit moeten gebeuren. Vanwaar deze check?

Als deze functie fout ging doordat `security_requirements` geen `dict` was, dan is er iets anders geks aan de hand en moet er iig iets gelogd worden.

Reply:
In de openapi heb ik bij de GET toegevoegd: 
      security:
      - OAuth2:
        - CAT/W
      -


Dan checkt het eerst of er een token is, maar als dat er niet is gaat het verder met security_requirements None . Ik weet niet hoe ik dat in de YAML spec moet toevoegen dat het een lege dictionary is . Misschien weet jij dat ?

Reply:
Ik heb het aangepast en het werkt inderdaad

Reply:
Oh, kan dat ook? Geweldig! Dat is inderdaad beter.